src/selectors/nodes.js

```diff
@@ -61,13 +67,18 @@ export const getNodeActive = createSelector(
       if (nodeID === hoveredNode) {
         return true;
       }
-      const activeViaTag = nodeTags[nodeID].some((tag) => tagActive[tag]);
-      const activeViaModularPipeline =
-        nodeModularPipelines[nodeID] &&
-        nodeModularPipelines[nodeID].some(
-          (modularPipeline) => modularPipelineActive[modularPipeline]
-        );
-      return Boolean(activeViaTag) || Boolean(activeViaModularPipeline);
+      const tags = nodeTags[nodeID] || [];
+      const activeViaTag = tags.some((tag) => tagActive[tag]);
+      const modularPipelines = nodeModularPipelines[nodeID] || [];
+      const activeViaModularPipeline = modularPipelines.some(
+        (modularPipeline) => modularPipelineActive[modularPipeline]
+      );
+      const isActiveModularPipeline = modularPipelineActive[nodeID];
+      return (
+        Boolean(activeViaTag) ||
```

Comment:
Probably don't need the `Boolean` for each of these as it's pretty clear and not used to cast (if it was used to cast though I'd say move the `Boolean` to the original assignment)